src/ImageSharp.Formats/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="ConfigurationExtensions.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp
+{
+    using System;
+    using System.IO;
+
+    using Formats;
+
+    /// <summary>
+    /// Extension methods for the <see cref="Configuration"/> type.
+    /// </summary>
+    public static partial class ConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds the common formats, PNG, JPEG, GIF and BMP.
+        /// </summary>
+        /// <param name="config">The config.</param>
+        /// <returns>The config</returns>
+        public static Configuration AddCommonFormats(this Configuration config)
```

Comment:
I'm happy with going another way with this, figured I would chuck it in and it could then be used as a bouncing off point tom do the rest... 👍 for only looking for core modules, we won't have the issue with loading everything up then and we wouldn't even need to scan the assembly for implementations of an interface as we should know exactly which classes in which assemblies we want.